scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -283,6 +291,56 @@ task AssertCostIsTrackedAndExpected {
     }
 
     output {
-      File cost_observability_output_csv = "cost_observability_output.csv"
+        File cost_observability_output_csv = "cost_observability_output.csv"
+    }
+}
+
+task AssertTableSizesAreExpected {
+    meta {
+        # we want to check the database each time this runs
+        volatile: true
+    }
+
+    input {
+        Boolean go = true
+        String dataset_name
+        String project_id
+        File expected_output_csv
+    }
+
+    command <<<
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        set -o xtrace
+
+        echo "project_id = ~{project_id}" > ~/.bigqueryrc
+        bq query --location=US --project_id=~{project_id} --format=csv --use_legacy_sql=false \
```

Comment:
TOL I think if you start the double quote on this line you could get rid of all the trailing backslashes. Maybe.